KubeArmor/eventAuditor/Makefile

```diff
@@ -0,0 +1,33 @@
+ARCH=$(shell uname -m)
```

Comment:
why is this Makefile required?
can you merge it into KubeArmor/Makefile?

Reply:
yes, sure